edflow/eval/pipeline.py

```diff
@@ -297,6 +297,11 @@ def after_epoch(self, epoch):
             kwargs = cb_kwargs.get(n, {})
             results[n] = cb(self.root, self.data_in, data_out, self.config, **kwargs)
         if self.clean_after_callbacks:
+            if not os.path.split(self.save_root)[1] == "model_outputs":
```

Comment:
Very nice!
Two ideas how to make this more robust:
1. use `os.path.basename(self.save_root)` to account for absolute paths.
2. use `self.save_root`.endswith("model_outputs")
I would prefer the first option.

Reply:
["`os.path.basename(path)`: Return the base name of pathname path. This is the second element of the pair returned by passing path to the function split()."](https://docs.python.org/3.7/library/os.path.html#os.path.basename) ;)

Reply:
Ooooh! I was thinking about `path.split('/')`. You are absolutely right of course!